.github/workflows/main.yml

```diff
@@ -15,13 +15,13 @@ jobs:
     if: github.event_name == 'push'
     steps:
       - uses: actions/checkout@v1
-      - uses: actions/setup-node@v1
-      - name: Tag using semantic release
+      - name: Bump version and push tag
+        id: version_bump
+        uses: anothrNick/github-tag-action@master
```

Comment:
Do we want to use `@master`? I suggest either similar to other actions, as in `@1.8` or the commit sha.

GitHub information about this:

> We strongly recommend that you include the version of the action you are using by specifying a Git ref, SHA, or Docker tag number. If you don't specify a version, it could break your workflows or cause unexpected behavior when the action owner publishes an update.
> 
>     Using the commit SHA of a released action version is the safest for stability and security.
>     Using the specific major action version allows you to receive critical fixes and security patches while still maintaining compatibility. It also assures that your workflow should still work.
>     Using the master branch of an action may be convenient, but if someone releases a new major version with a breaking change, your workflow could break.

Reply:
Agree. It's actually `@1.8.0`.

I copied 1.8 without checking :man_facepalming:

Reply:
1.8 will allow for automatic minor bump?

Reply:
there is no 1.8 tag, you need to provide an existing tag

Reply:
Oh, didn't know 😅 thought it would work like they say in the docs:

> Using the specific major action version allows you to receive critical fixes and security patches while still maintaining compatibility. It also assures that your workflow should still work.

Reply:
Ah, I see. I seems the doc implies that, however when I tried it failed :(

Reply:
Thank you! Changed it now